src/Zilliqa/Address.cpp

```diff
@@ -29,6 +30,13 @@ Address::Address(const PublicKey& publicKey) {
     std::copy(data.end() - Address::size, data.end(), bytes.begin());
 }
 
+Address::Address(const Data& data) {
+    if (!isValid(data)) {
+        throw std::invalid_argument("Invalid address data");
```

Comment:
can we avoid throwing here? per https://github.com/TrustWallet/wallet-core/issues/424

Reply:
This method is not exposed to C interface, only used in tests